doc/intro.rst

```diff
@@ -35,14 +35,38 @@ some thing to keep in mind:
 Installation
 ------------
 
-The `python-control` package may be installed using pip or the standard
-distutils/setuptools mechanisms.
+The `python-control` package may be installed using pip, conda or the
+standard distutils/setuptools mechanisms.
 
 To install using pip::
 
   pip install slycot   # optional
   pip install control
 
+Many parts of `python-control` will work without `slycot`, but some
+functionality is limited or absent, and installation of `slycot` is
+recommended.  
+
+*Note*: the `slycot` library only works on some platforms, mostly
+linux-based.  Users should check to insure that slycot is installed
+correctly by running the command::
+
+  python -c "import slycot"
+
+and verifying that no error message appears.  It may be necessary to install
+`slycot` from source, which requires a working FORTRAN compiler and the
+`lapack` library.  More information on the slycot package can be obtained
+from the `slycot project page <https://github.com/python-control/Slycot>`_.
+
+For users with a working the Anaconda distribution of Python, the following
```

Comment:
The change is trivial, if the text should indeed be different, and can be applied directly to `master` branch.